packages/flutter/lib/src/material/material_state.dart

```diff
@@ -233,6 +233,15 @@ class _MaterialStateColor extends MaterialStateColor {
 ///
 /// This should only be used as parameters when they are documented to take
 /// [MaterialStateMouseCursor], otherwise only the default state will be used.
+///
+/// This class also predefines several kinds of material state mouse cursors.
+///
+/// See also:
+///
+///  * [MouseCursor] for introduction on the mouse cursor system, such as how to
+///    use them.
+///  * [SystemMouseCursors], which contains many cursors supported natively by the
```

Comment:
which defines cursors that are supported by most native platforms

Reply:
I wonder what your version means, because:
1. If it means some platforms do not support system mouse cursors, then they do not support mouse cursors. There hasn't been a platform that support mouse cursors that do not come with some system cursors.
2. If it means these cursors are only the most common ones, it's quite the opposite. `SystemMouseCursors` is meant to be a superset of all cursors from all platforms. Some of these cursors are only supported by 1 platform, we include them anyway, and they fall back to the basic cursor on other platforms.

Reply:
I agree that native platform is redundant.

By "cursors supported by most platforms" I was implying that the superset of cursors we picked for SystemMouseCursors _mostly_ included cursors that were supported in multiple platforms. Otherwise developers could reasonably ask if we were really balancing portability with flexibility.